lib/glific_web/schema.ex

```diff
@@ -66,6 +67,8 @@ defmodule GlificWeb.Schema do
 
     import_fields(:webhook_log_queries)
 
+    import_fields(:notification_queries)
```

Comment:
Let's add them in the import order. Though it does not have any impact, I just want to have consistency.